tuf/encoding/metadata_definitions.asn1

```diff
@@ -0,0 +1,302 @@
+/*
+This file defines the ASN.1 format for TUF metadata in a language-neutral,
+platform-neutral manner.
+
+This is not the *internal* format used in this TUF implementation, but it is
+supported. Please see the TUF specification for the JSON format and more
+detailed explanation of TUF metadata.
+
+The TUF implementation in this repository uses modules in tuf.encoding and
+the Python library pyasn1 to construct, convert, encode, and decode metadata
+using ASN.1 and ASN.1/DER.
+
+
+NO: CURRENTLY SKIPPING THIS STEP:
+  We compile this abstract file into a slightly lower level version of this
+  file (with IMPLICIT markers and sequence positions) using asn1c and then
+
+We convert this file into a Python module, metadata_asn1_definitions.py, which
+pyasn1 and those modules can use. The instructions for doing this are included
+at the top of metadata_asn1_definitions.py
+
+
+This file is organized into five sections:
+   Common Definitions
+   Root Role Definitions
+   Timestamp Role Definitions
+   Snapshot Role Definitions
+   Targets Role Definitions
+
+
+A note for implementers of TUF-specification-conformant systems:
+There are constraints and optimizations that you can bake into your ASN.1
```

Comment:
Was this meant to be "These are..."? If so, a better wording might be "The following are..."

Reply:
No, it's intended to be "There are", i.e. "There exist". I'll reword it to make it clearer.